src/platform/plugins/shared/workflows_management/server/workflow_event_logger/index_mappings.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import type { MappingTypeMapping } from '@elastic/elasticsearch/lib/api/types';
+
+export const WORKFLOW_EXECUTION_LOGS_INDEX_MAPPINGS: MappingTypeMapping = {
```

Comment:
many of these mappings should be already covered by `ecs@mappings` component template https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/template-resources/src/main/resources/ecs%40mappings.json , consider using this component template instead of explicitly mapping ECS fields

Reply:
more info on component templates  https://www.elastic.co/observability-labs/blog/future-proof-your-logs-with-ecs-mappings-template , can ofc be smth for the future

Reply:
we will address it in the future! adding a comment about it